github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -2341,6 +2341,53 @@ repositories:
         dismissalRestrictions:
           - sigstore-rs-codeowners
     webCommitSignoffRequired: true
+  - name: sigstore-ruby
```

Comment:
this will be a new repo? i opened and that redirect to `ruby-sigstore`

let sync that, maybe need some manual operations